back/auth.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Body, Controller, Post } from "@nestjs/common";
+import { AuthService } from "./auth.service";
+import { AuthDto } from "./dto";
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @Post('signup')
+  signup(@Body() dto: AuthDto) {
+    return this.authService.signup(dto);
+  }

```

Comment:
attention aux indentations
```suggestion
    @Post('signup')
    signup(@Body() dto: AuthDto) {
      return this.authService.signup(dto);
    }

```